doc/development/adding_templates.rst

```diff
@@ -14,14 +14,15 @@ PennyLane's :mod:`template <pennylane.template>` library.
     :mod:`subroutines <pennylane.templates.subroutines>`. Below you need to replace ``<templ_type>`` with the
     correct template type.
 
-Templates are just gates
-~~~~~~~~~~~~~~~~~~~~~~~~
+Templates are just operations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Conceptually, there is no difference in PennyLane between a template or ansatz and a :doc:`operation </introduction/operations>`.
-Both inherit from the :class:`Operation <pennylane.operation.Operation>` class, which has an ``expand()`` function
-that can be used to define a decomposition into other gates. If a device does not recognise the name of the operation,
-it calls the ``expand()`` function which returns a :class:`tape <pennylane.tape.QuantumTape>` instance that
-represents the queue of the decomposing gates.
+Conceptually, there is no difference in PennyLane between a template or ansatz and a gate - they are
+both :doc:`operations </introduction/operations>` and inherit from the
+:class:`Operation <pennylane.operation.Operation>` class. Unless a device knows how to implement this class on a
+quantum computer, it queries the operation's ``expand()`` method, which expresses the operation as a
+decomposition of other operations. More precisely, this method returns a :class:`tape <pennylane.tape.QuantumTape>`
+instance that represents the queue of the decomposition.
```

Comment:
I sneaked in a little overhaul of this file...

Reply:
> Conceptually, there is no difference in PennyLane between a template or ansatz and a gate - they are

This is great to have! I have a received a lot of questions regarding the difference lately.